boilerplate/openshift/golang-codecov/.codecov.yml

```diff
@@ -0,0 +1,26 @@
+codecov:
```

Comment:
The objective of this first step is to make a simple/standalone convention, hence the copied files. Once strategy for golang-osd-operator is defined, it will cover the clean-up of those dupps ( [OSD-10468](https://issues.redhat.com/browse/OSD-10468) )